content/en/docs/plugin-guide/plugin-policy-engine.md

```diff
@@ -0,0 +1,450 @@
+---
+title: Policy Engine Plugin
+toc_hide: true
+exclude_search: true
+description: >
+  The Policy Engine plugin is the next iteration of Armory's Policy Engine for Spinnaker.
+---
+<!-- This plugin is the next iteration of our Policy Engine extension and is not ready for public consumption. This unlisted page is to satisfy an auditing requirement that one of our customers has. It is also hidden via robots.txt and the netlify sitemap plugin. -->
+
+## Overview
+
+Armory's Policy Engine plugin is the next iteration of the Policy Engine feature that ships with our Armory distribution. Not only does it include support for existing features like applying policy to pipelines as they're being saved, it also introduces policy hooks into other services like Gate and Orca. 
+
+## Requirements
+
+This plugin requires:
+
+- Armory 2.23.x or later (OSS Spinnaker 1.23.x or later)
+- Open Policy Agent 0.12.x or later
+
+<!--## Limitations
+
+TODO - figure out if there are any limitations
+-->
+
+## Setup
+
+The plugin can be delivered using two different methods:
+
+1. Docker image as an init container on each affected service
+1. Using a remote plugin repository
```

Comment:
Shouldn't this be a `2`?
```suggestion
2. Using a remote plugin repository
```

Reply:
The site generator auto increments numbered lists, so you could just do 1s all the way down. But yeah, it looks weird to humans when writing it lol.

Reply:
Actually, Markdown will still render it properly. One reason to just use `1.` is so you don't have to re-number the whole list when adding a new item. :D